stacks/SAWO/deploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+set -e
+
+################################################################################
+# repo
+################################################################################
+helm repo add stable https://charts.helm.sh/stable
+helm repo update > /dev/null
+
+################################################################################
+# chart
+################################################################################
+STACK="SAWO"
```

Comment:
Please change all instances of `SAWO` to `sawo`, including the directory name. Otherwise, the installation wont work.

Reply:
@paulanunda I've done the renaming in all the files but I'm unable to rename the directory.